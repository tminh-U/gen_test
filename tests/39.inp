15
19 20 10 17 7 3 4 18 19 11 16 8 7 8 1 
15
14 15
1 14
9 9
10 14
10 10
1 4
7 13
8 10
2 8
5 6
1 2
6 12
10 15
1 15
2 14
