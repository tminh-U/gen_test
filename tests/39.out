3
79
2
9
2
7
18
5
17
3
4
32
10
80
76
